src/units/index.ts

```diff
@@ -99,7 +99,12 @@ export const current = z
   .or(z.number())
   .transform((v) => parseAndConvertSiUnit(v).value!)
 
-export const time = z
```

Comment:
I dont think we should break any compat here, we should export time as an alias for duration